src/scripts/templates/variant/Input Vector default template.js

```diff
@@ -41,19 +45,17 @@ function parseParameters(helper, msg) {
 
 function setParameter(helper, msg, param, value, escaped) {
     size = helper.getParamNumber();
-    if (size > 1) {         
-        query = helper.getParamValue(size);
-        
-        for (var i = 0; i < size; i++) {
-            pname = getParamName(i);
-            pvalue = getParamValue(i);                
-            if (pname == param) {      
-                    pvalue = helper.encodeBase64(value);
-            }
-                    
-            query = pname + "=" + pvalue + "&" + query;
+    query = helper.getParamValue(size);
```

Comment:
Just realised that this should be `size - 1`, to obtain the `notb64` parameters.
Also, in the statement:
```JavaScript
nob64 = nob64 + "&" + vars;
```
it should be:
```JavaScript
nob64 = nob64 + "&" + vars[i];
```
(to just add the current parameter, not all of them)

If you are OK changing the other semantic issues I have other suggestions (like use a local variable, instead of adding "helper" parameter).

Reply:
Sure, just let me know.